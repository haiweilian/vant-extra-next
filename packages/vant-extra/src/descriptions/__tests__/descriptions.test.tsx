import { nextTick } from 'vue'
import { mount } from '@vue/test-utils'
import { describe, test, expect } from 'vitest'
import { Descriptions } from '../index'

describe('Descriptions', () => {
  test('render', async () => {
    const wrapper = mount(Descriptions)
    await nextTick()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
