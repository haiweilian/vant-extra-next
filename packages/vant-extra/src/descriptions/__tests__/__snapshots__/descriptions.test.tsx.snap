// Vitest Snapshot v1

exports[`Descriptions > render 1`] = `
"<div class=\\"vae-descriptions vae-descriptions--colon vae-descriptions--normal\\">
  <!---->
  <table class=\\"vae-descriptions__table\\">
    <tbody></tbody>
  </table>
</div>"
`;
